blender/arm/logicnode/animation/LN_action.py

```diff
@@ -4,8 +4,10 @@ class AnimActionNode(ArmLogicTreeNode):
     """Anim action node"""
     bl_idname = 'LNAnimActionNode'
     bl_label = 'Action'
+    arm_version = 1
 
     def init(self, context):
+        super(AnimActionNode, self).init(context)
```

Comment:
Out of couriosity: I think those lines can be simplified to `super().init(context)` since Python 3 or is there a reason behind this?

Reply:
yes, they can. and no, I don't have a reason to choose this, really.
maybe because it is easier to understand this form for people who never saw a super() before?